client/components/Reports/AriaSupportTable.jsx

```diff
@@ -42,15 +46,25 @@ const AriaSupportTable = ({ ariaHtmlFeaturesMetrics }) => {
           {keys.map(key => (
             <tr key={key}>
               <td>
-                <Link href={links[key].href}>{key}</Link>
+                <a href={links[key].href} rel="noreferrer" target="_blank">
+                  {key}
+                </a>
               </td>
-              {supportCombos.map(col => (
-                <td key={key + col}>
-                  <ProgressBar
-                    progress={dataByFeature[key][col].passedPercentage}
-                  />
-                </td>
-              ))}
+              {supportCombos.map(col => {
+                const row = dataByFeature[key][col];
+                const ids = atBrowserLinks[key][col];
```

Comment:
Not a huge deal, but I don't think you need `atBrowerLinks` as it just copies values from `row` which you get from `dataByFeature` here